components/list/demo/list-item-sample.js

```diff
@@ -30,17 +50,46 @@ class ListItemSample extends ListItemCheckboxMixin(LitElement) {
 				` : nothing }
 				${this.selectable ? html`
 				<div slot="control">${ this._renderCheckbox() }</div>
-				<div slot="control-action">${ this._renderCheckboxAction() }</div>
+				<div slot="control-action" aria-labelledby="${this._contentId}">${ this._renderCheckboxAction() }</div>
+				` : nothing }
+				${ this.href ? html`
+				<a slot="content-action"
+					class="d2l-list-item-link"
+					href="${this.href}"
+					aria-labelledby="${this._contentId}"
+					@mouseover="${this._anchorEventHandler}"
+					@mouseout="${this._anchorEventHandler}"
```

Comment:
Arrow functions are not allowed in the eslint, which was my only real reasoning for making the shared handler. Is it better to have four separate handlers in this case?

Reply:
The linter was hating on `@mouseenter="${() => this._hovering = true;}"` is that because we don't need the `${}` part? That would be awesome for speed.

Reply:
I see, the rationale can be found [here](https://github.com/43081j/eslint-plugin-lit/blob/master/docs/rules/no-template-arrow.md).  I was wondering if that were the case... indeed.   This is just a sample, but if it were me I would define 4 different handlers.  Once the `_hovering` and `_focusing` properties are defined, then handlers just become:
```
_handleMouseEnter() { this._hovering = true; }
_handleMouseLeave() { this._hovering = false; }
etc.
```

Reply:
They make it seem so dire. It isn't every time render happens. It is every time that part gets render. Even then, the defaultRenderProcessor might cache it. Anyways, easier to define the smaller method then going into the lower levels and check it out.